src/app/services/media-to-grid-media.service.ts

```diff
@@ -20,35 +21,34 @@ export class MediaToGridMediaService {
     }
   }
 
-  private personMapper(person: Person): GridMediaItem {
+  private personMapper(person: Person | Cast): GridMediaItem {
     const { id, name, photo, mediaType } = person;
     return {
       id,
       title: name,
       type: mediaType,
       image: photo,
+      subtitle: 'character' in person ? person.character : undefined,
```

Comment:
Maybe there is a better way to do it?

Reply:
I believe better to use another mapper. CastMapper or something like that. You can reuse one mapper inside other.

```
  private castMapper(cast: Cast): GridMediaItem {
    return {
      ...personMapper(cast),
      subtitle: cast.character
    }
  }
```

Reply:
Moved check to typeGuard, thanks.